### PR TITLE
[15/3] Update DBC billing and subscription

### PR DESCRIPTION
**Pull request by stephaurelio**
## Description

This PR updates the billing and subscription docs for DBC. It also restructures the subscription content to incorporate a new page for managing seats and users.

- [Billing deploy preview](https://deploy-preview-19623--docsdocker.netlify.app/billing/build-billing/)
- [Subscription deploy preview](https://deploy-preview-19623--docsdocker.netlify.app/subscription/build-cloud/build-details/) 
- [Seats deploy preview](https://deploy-preview-19623--docsdocker.netlify.app/subscription/build-cloud/manage-seats/)

## Related issues or tickets

Closes JIRA: https://docker.atlassian.net/browse/ENGDOCS-2011 

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

File: content/subscription/build-cloud/manage-seats.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+You can add seats and manage invitations to your Docker Build Cloud Team in the Docker Build Cloud dashboard.
+
+> **Note**
+>
+>If you have a Docker Build Cloud Business subscription, you can add and remove seats by working with your account executive, then assign your purchased seats in the Docker Build Cloud dashboard.
+
+## Add seats
+
+1. Sign in to your [Docker account](https://login.docker.com/) and go to [Docker Build Cloud](https://build.docker.com/).
```
Comment (dvdksn) [inline content/subscription/build-cloud/manage-seats.md:15]: suggestion: Shouldn't it be "Go to Docker Build Cloud and sign in to your account"? Here and in the other step 1's. Do they need to take a detour via login.docker.com?

Comment (dvdksn) [reply content/subscription/build-cloud/manage-seats.md:15]: It we do need the detour (signing in via a separate portal before going to the DBC dashboard, we should make it a separate step

File: content/subscription/build-cloud/manage-seats.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+2. Select **Settings & billing**.
+3. Under your current plan, select **Add seats**.
+4. Select the number of seats you want to add, then select **Add seats**.
+
+The number of seats will be charged to your payment information on file, and are added immediately.
+
+## Remove seats
+
+1. Sign in to your [Docker account](https:/hub.docker.com/login) and go to [Docker Build Cloud](https://build.docker.com/).
```
Comment (dvdksn) [inline content/subscription/build-cloud/manage-seats.md:24]: The other step linked to login.docker.com, this one links to hub.docker.com/login

File: content/subscription/build-cloud/manage-seats.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+3. Under your current plan, select **Remove seats**.
+4. Enter the total number of seats you want to keep on your plan, then select **Remove seats**.
+
+The charge for the reduced seat count will be reflected on the next billing cycle. Optionally, you can cancel the seat downgrade any time before the next billing cycle.
+
+## Invite users
+
+As an owner of the Docker Build Cloud team, you can invite members to access cloud builders. To invite team members to your team in Docker Build Cloud:
+
+1. Sign in to your [Docker account](https:/hub.docker.com/login) and go to [Docker Build Cloud](https://build.docker.com/).
```
Comment (dvdksn) [inline content/subscription/build-cloud/manage-seats.md:35]: same as the other comments

File: content/subscription/build-cloud/manage-seats.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+2. Select **User management**.
+3. Select **Invite**.
+4. Enter the email address or Docker ID of the member you want to invite. If the invitee doesn’t have a Docker ID, you need to enter their email address so they can create an account when they accept the invite.
```
Comment (dvdksn) [inline content/subscription/build-cloud/manage-seats.md:38]:
```suggestion
4. Enter the email address or Docker ID of the member you want to invite. If the invitee doesn’t have a Docker ID, enter their email address so they can create an account when they accept the invite.
```

Comment (dvdksn) [reply content/subscription/build-cloud/manage-seats.md:38]: if the user has a Docker ID, can you just enter the email anyway? Or do you have to enter it differently depending on if they already have an account or not?

Comment (stephaurelio) [reply content/subscription/build-cloud/manage-seats.md:38]: Yeah, you can do email either way but if there's no Docker ID it has to be email. I can tweak this wording!

File: content/subscription/build-cloud/manage-seats.md (hunk 2 above)
Comment (stephaurelio) [reply content/subscription/build-cloud/manage-seats.md:24]: Ah that slipped in it's the wrong URL. But anyway I'll update these to just sign in to Docker Build Cloud!

**Review by dvdksn — APPROVED** [review]
LGTM!

**Pull request merged**